Documentation/Functions/Data.rst

```diff
@@ -797,6 +797,45 @@ Get content from the :ref:`register <using-setting-register>`:
 
     lib.foo.data = register : color
 
+
+..  _data-type-gettext-request:
+
+request
+-------
+
+..  versionadded:: 12.4
+
+..  confval:: request
+    :name: data-request
+    :Syntax: request : [attribute] | [property]
+
+    Retrieve the property of a
+    :ref:`PSR-7 request attribute <t3coreapi:request-attributes>`.
+
+    Note that only scalar properties can be retrieved: int, float, string or
```

Comment:
Note to reviewers: It is intended not to use an admonition to avoid box in box visually, which looks cluttered.

Reply:
Agreed!